plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/block.json

```
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"name": "woocommerce/product-filter-attribute",
 	"version": "1.0.0",
-	"title": "Filter Options",
+	"title": "Attribute",
```

Should we use the `Attribute (Experimental)` as before?

```suggestion
	"title": "Attribute (Experimental)",
```